tests/test_sqlite.py

```diff
@@ -6,6 +6,7 @@
 
 from pydanticrud import BaseModel, SqliteBackend
 from rule_engine import Rule
+from pydantic import ConfigDict
```

Comment:
Doesn't look like we need to import this anymore.

Reply:
Removed